<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@3.4">
  <node class="cps.n.StepAtomNode" plugin="workflow-cps@2.80">
    <parentIds>
      <string>8</string>
    </parentIds>
    <id>9</id>
    <descriptorId>org.jenkinsci.plugins.workflow.steps.durable_task.ShellStep</descriptorId>
  </node>
  <actions>
    <cps.a.ArgumentsActionImpl plugin="workflow-cps@2.80">
      <arguments>
        <entry>
          <string>script</string>
          <wf.a.ArgumentsAction_-NotStoredReason>MASKED_VALUE</wf.a.ArgumentsAction_-NotStoredReason>
        </entry>
      </arguments>
      <isUnmodifiedBySanitization>false</isUnmodifiedBySanitization>
    </cps.a.ArgumentsActionImpl>
    <wf.a.TimingAction plugin="workflow-api@2.40">
      <startTime>1587708484456</startTime>
    </wf.a.TimingAction>
    <s.a.LogStorageAction/>
    <wf.a.ErrorAction plugin="workflow-api@2.40">
      <error class="hudson.AbortException">
        <detailMessage>script returned exit code 127</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.handleExit(DurableTaskStep.java:658)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:604)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:548)</trace>
          <trace>java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)</trace>
          <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)</trace>
          <trace>java.lang.Thread.run(Thread.java:748)</trace>
        </stackTrace>
        <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" resolves-to="java.util.Collections$UnmodifiableList">
          <c class="list"/>
          <list reference="../c"/>
        </suppressedExceptions>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>